src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write code here
+const lilist = document.querySelectorAll('li');
```

Comment:
this name is so complex, please rename it to be: `const listItems = document.querySelectorAll('li');`

it will be more readable and easier to read 😃